### PR TITLE
Clockcult is penalized for reciting scripture offstation

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Reciting scripture while not on the station, centcom, or mining/lavaland will double recital time and component costs.
/:cl:

"But this isn't that much", you say. Spending twice and many components and twice as much time for anything offstation is massive, especially given conversion is affected by this.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Do solars count as on or off station

**Comment by ChangelingRain** [comment]
On station but not particularly practical
most humans need air and clockwork armor isn't very good against pressure

**Comment by RemieRichards** [comment]
I wouldn't grant lavaland that freedom, it's already a hotspot cult base, don't make it more mechanically viable.

**Comment by Tacolizard** [comment]
this kills tactical spacebases. seriously spacebases were a really cool aspect of cuck cult, and why would you penalize this for any other reason than nerfing that? is there some other reason for this?

**Comment by ExcessiveUseOfCobblestone** [comment]
you can still make space bases, they just have to be on the station Zlevel

**Comment by Anonmare** [comment]
Eh, I'm fine with it. Don't particularly care either way tbh

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 1]
```diff
@@ -35,8 +35,19 @@ Judgement: 12 servants, 5 caches, 300 CV, and any existing AIs are converted or
...
+	var/static/list/inathneq_penalty = list("Child, this is too far out!", "The barrier isn't thin enough for for me to help!", "Please, go to the station so I can assist you.", \
+	"Don't waste my Cogs on this...", "There isn't enough time to linger out here!")
+	var/static/list/sevtug_penalty = list("Fool! Get to the station and don't waste capacitors.", "You go this far out and expect help?", "The veil is too strong, idiot.", \
+	"How does the Justicar get anything done with servants like you?", "Oh, you love wasting time, don't you?")
+	var/static/list/nezbere_penalty = list("You disgrace our master's name with this endeavour.", "This is too far from the station to be a good base.", "This will take too long, friend.", \
+	"The barrier isn't weakened enough to make this practical.", "Don't waste alloy.")
+	var/static/list/nzcrentr_penalty = list("You'd be easy to hunt in that little hunk of metal.", "Boss says you need to get back to the beacon.", "Boss says I can kill you if you do this again.", \
+	"Sending you power is too difficult here.", "Boss says stop wasting time.")
+
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: ❤️ it

**Review by ExcessiveUseOfCobblestone — APPROVED** [review]

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 2]
```diff
@@ -35,8 +35,19 @@ Judgement: 12 servants, 5 caches, 300 CV, and any existing AIs are converted or
...
+	var/static/list/inathneq_penalty = list("Child, this is too far out!", "The barrier isn't thin enough for for me to help!", "Please, go to the station so I can assist you.", \
+	"Don't waste my Cogs on this...", "There isn't enough time to linger out here!")
+	var/static/list/sevtug_penalty = list("Fool! Get to the station and don't waste capacitors.", "You go this far out and expect help?", "The veil is too strong, idiot.", \
+	"How does the Justicar get anything done with servants like you?", "Oh, you love wasting time, don't you?")
+	var/static/list/nezbere_penalty = list("You disgrace our master's name with this endeavour.", "This is too far from the station to be a good base.", "This will take too long, friend.", \
+	"The barrier isn't weakened enough to make this practical.", "Don't waste alloy.")
+	var/static/list/nzcrentr_penalty = list("You'd be easy to hunt in that little hunk of metal.", "Boss says you need to get back to the beacon.", "Boss says I can kill you if you do this again.", \
+	"Sending you power is too difficult here.", "Boss says stop wasting time.")
```
Comment (Iamgoofball) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: He's coming too?

**Pull request merged**